src/tests/request.model.spec.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable max-len */
+import chai from 'chai';
+import db from '../database/models';
+
+const { expect } = chai;
+
+describe('request model tests', () => {
+  it('It should return all two way trip with length of two', async () => {
+    const trips = await db.Trips.count({ include: [{ model: db.Requests, where: { returnTime: { [db.Sequelize.Op.not]: null } } }] });
```

Comment:
Good job @urbanishimwe  but I don't think you should disable Eslint rule I think you should format your code according to the rule

Reply:
If i do hound will fail